### DISCUSSION TITLE
Will Zed have bimap font support?

### DISCUSSION THREAD

**Discussion by JackNWhite** [zed-industries/zed#8017] [Feature Requests]
I am a Terminus zealot; will I be able to use PCF or OTB fonts in Zed? I do not have macOS, which is why I ask.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
